server/sonar-server/src/test/java/org/sonar/server/issue/ws/SearchActionMediumTest.java

```diff
@@ -446,7 +446,7 @@ public void filter_by_assigned_to_me() throws Exception {
       .setStatus("OPEN")
       .setKee("82fd47d4-b650-4037-80bc-7b112bd4eac2")
       .setSeverity("MAJOR")
-      .setAssignee("john");
+      .setAssignee("john-bob.polop");
```

Comment:
I had an exception when clicking on my name using dory-latest; turns out it was the ``-`` in the middle of my login which triggered a parse error in ES.